Cargo.toml

```diff
@@ -42,6 +42,7 @@ crossterm = { version = "0.25", optional = true }
 strum = "0.24"
 strum_macros = "0.24"
 unicode-width = "0.1"
+console = "0.15.1"
```

Comment:
`console` has a `unicode-width` feature flag.

Without this, multi-character unicode symbols won't be detected as such.